website/conferences/message.py

```diff
@@ -164,3 +160,11 @@ def attachments(self):
                 range(count),
             ),
         )
+
+    @property
+    def find_allowed_types(self):
+        from .model import Conference
+        all_confs = list(Conference.find())
+        allowed_types = [conf.field_names['talk'] for conf in all_confs] + [conf.field_names['poster'] for conf in all_confs]
```

Comment:
This function needs to be updated for the change in field_names. It may be the origin of the failing unit tests.